lib/claim_letters/claim_letter_downloader.rb

```diff
@@ -6,6 +6,20 @@ module ClaimStatusTool
   class ClaimLetterDownloader
     FILENAME = 'ClaimLetter'
     DEFAULT_ALLOWED_DOCTYPES = %w[184].freeze
+    DOCTYPE_TO_DISPLAY_DESCRIPTION = {
```

Comment:
This is the same mapping from the frontend, which also mirrors Julie's table in the initial change request.